MIGRATION.md

```diff
@@ -230,9 +233,27 @@ This was done to remove a hard dependency on `https-proxy-agent` when running
    authorization: 'authorization'
  }`
 
-### 8. Bug fix: Setting an explicit `algorithm` when generating a wallet works now
+### 8. `Wallet` functions default to `ed25519` instead of `secp256k1` signing algorithm
+
+With 3.0, we updated the default signing algorithm used by the Wallet object from secp256k1 to ed25519 in order to default to the higher-performance algorithm. This is a breaking change to all functions used to generate a Wallet, so if you have a pre-existing XRPL account that you're using to generate a specific Wallet using older versions of xrpl.js, you must specify that you are using secp256k1 as the algorithm to decode your private key / seed / etc to get the same behavior as before. See below for specifically how to update your code.
+
+If you are creating new accounts each time (ex. via `Client.fundWallet`), you do not need to specify the signing algorithm.
+
+**Before**
+
+`Wallet.fromSeed('s...')`
+`Wallet.generate()`
```

Comment:
`Wallet.generate` users don't need to change.

Reply:
yes, although the internals of the `Wallet.generate` has been updated to incorporate the `algorithm` parameter (PR: https://github.com/XRPLF/xrpl.js/commit/041055082a6f689ea30dcc1e1a04aba6103f537b), the external behavior hasn't changed (as evidenced by the un-modified `Wallet.generate` unit tests)

Reply:
fixed in [a16773e](https://github.com/XRPLF/xrpl.js/pull/2659/commits/a16773ee21847ed903fca9f0c6b77098890201fb)